src/server.c

```diff
@@ -3723,7 +3717,11 @@ void call(client *c, int flags) {
     dirty = server.dirty;
     prev_err_count = server.stat_total_error_replies;
 
-    updateCachedTime(0);
+    /* Update cache time, in case we have nested calls we want to
+     * update only on the first call*/
+    if (server.fixed_time_expire++ == 1) {
```

Comment:
Maybe it was my misleading comment. But that's a bug.. Either use 0 or ++ before the var.. 
```suggestion
    if (server.fixed_time_expire++ == 0) {
```